Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -265,7 +266,9 @@ def test_nonascii(self):
             with self.subTest(name):
                 module = self.load_module_by_name(name)
                 self.assertEqual(module.__name__, name)
-                self.assertEqual(module.__doc__, "Module named in %s" % lang)
+                if HAVE_DOCSTRINGS:
+                    self.assertEqual(module.__doc__,
+                                     "Module named in %s" % lang)
```

Comment:
For readability and consistency, I would recommend replacing the old string formatting with f strings:
```python
"Module named in %s" % lang
f"Module named in {lang}"
```
This does not make a functional difference, but since this section is being modified (and it's the only section in the file which is using the old string formatting) it may as well be updated to use f strings instead.